source/dotnet/wholesale-api/Edi.UnitTests/AggregatedTimeSeriesRequestHandlerTests.cs

```diff
@@ -276,6 +290,61 @@ public class AggregatedTimeSeriesRequestHandlerTests
             Times.Once);
     }
 
+    [Theory]
+    [InlineAutoMoqData]
+    public async Task ProcessAsync_WhenNoCalculations_SendsRejectedEdiMessage(
+        [Frozen] Mock<IAggregatedTimeSeriesQueries> aggregatedTimeSeriesQueries,
+        [Frozen] Mock<IEdiClient> senderMock,
+        [Frozen] Mock<IValidator<AggregatedTimeSeriesRequest>> validator,
+        [Frozen] Mock<ILogger<AggregatedTimeSeriesRequestHandler>> logger,
+        [Frozen] Mock<CompletedCalculationRetriever> completedCalculationRetriever)
+    {
+        // Arrange
+        const string expectedRejectedSubject = nameof(AggregatedTimeSeriesRequestRejected);
+        var expectedReferenceId = Guid.NewGuid().ToString();
+        var request = AggregatedTimeSeriesRequestBuilder
+            .AggregatedTimeSeriesRequest()
+            .WithRequestedByActorRole(DataHubNames.ActorRole.EnergySupplier)
+            .WithGridArea("303")
+            .Build();
+        var serviceBusReceivedMessage = ServiceBusModelFactory.ServiceBusReceivedMessage(
+            properties: new Dictionary<string, object> { { "ReferenceId", expectedReferenceId } },
+            body: new BinaryData(request.ToByteArray()));
+
+        var start = DateTimeOffset.Parse("2022-01-01T00:00Z").ToInstant();
+        var end = DateTimeOffset.Parse("2022-01-01T00:15Z").ToInstant();
+        var aggregatedTimeSeries = CreateAggregatedTimeSeries(start, end);
```

Comment:
```suggestion
        var start = DateTimeOffset.Parse("2021-12-31T23:00:00Z").ToInstant();
        var end = DateTimeOffset.Parse("2022-01-31T23:00:00Z").ToInstant();
        var aggregatedTimeSeries = CreateAggregatedTimeSeries(start, end);
```
Not that it matters, but this is not a valid request. Since it would be rejected by a validation rule